pkg/api/types.go

```diff
@@ -1980,3 +1980,21 @@ type MetadataWithTest struct {
 func (m *MetadataWithTest) JobName(prefix string) string {
 	return m.Metadata.JobName(prefix, m.Test)
 }
+
+type Architecture string
+
+const (
+	AMD64Arch Architecture = "amd64"
+	ARM64Arch Architecture = "arm64"
+)
+
+var archToCluster = map[Architecture]Cluster{
```

Comment:
```suggestion
var archToCluster = map[Architecture][]Cluster{
```

But this is not important now. Once we will add the other clusters, we can make the change. Feel free not to address this or write a comment